packages/app/src/app/pages/Dashboard/Content/routes/Settings/components/PermissionSettings.tsx

```diff
@@ -118,8 +118,9 @@ const MinimumPrivacy = ({ disabled }: { disabled: boolean }) => {
       gap={114}
       css={css({
         padding: 6,
+        backgroundColor: '',
```

Comment:
Is this needed? I believe it is parsed out when the css is computed

Reply:
✅